pkg/services/authn/authnimpl/service.go

```diff
@@ -62,6 +62,7 @@ func ProvideService(
 	features *featuremgmt.FeatureManager, oauthTokenService oauthtoken.OAuthTokenService,
 	socialService social.Service, cache *remotecache.RemoteCache,
 	ldapService service.LDAP, registerer prometheus.Registerer,
+	keyService auth.KeyService,
```

Comment:
We don't need to inject it here

Reply:
Where does it needs to be injected?

Reply:
For now nowhere, we don't have any consumers of this service yet :)

Reply:
Removed